library/ecp_internal.h

```diff
@@ -28,6 +28,18 @@
 #include "mbedtls/bignum.h"
 #include "bignum_mod.h"
 
+/** Convert an MPI to its canonical representative.
+ *
+ * \note Currently handles the case when `N->int_rep` is
+ * MBEDTLS_MPI_MOD_REP_MONTGOMERY.
```

Comment:
This note is not needed: Montgomery representation only affects multiplication and inversion (exponentiation). Subtraction and addition work the same in either representation.